packages/common/src/store/account/slice.ts

```diff
@@ -137,6 +132,18 @@ const slice = createSlice({
       }>
     ) => {
       state.guestEmail = action.payload.guestEmail
+    },
+    updatePlaylistLibrary: (state, action: PayloadAction<PlaylistLibrary>) => {
+      state.playlistLibrary = action.payload
+    },
+    incrementTrackSaveCount: (state) => {
```

Comment:
i wonder why we even need this at all? is it for the profile completion meter? your user has repost count on it https://discoveryprovider.audius.co/v1/users/handle/rayjacobson it should probably have save count too? just thinking aloud....

Reply:
It gets used in certain places as a limit amount for fetching things. I agree that I think there's probably a better way to do this (like a count endpoint on library/tracks).